instructors.md

```diff
@@ -570,51 +591,50 @@ Solutions to exercises:
 >
 > Write a command-line program that does addition and subtraction:
 >
-> ~~~ {.python}
+> ~~~ {.bash}
 > $ python arith.py add 1 2
 > ~~~
 > ~~~ {.output}
 > 3
 > ~~~
-> ~~~ {.python}
+> ~~~ {.bash}
 > $ python arith.py subtract 3 4
 > ~~~
 > ~~~ {.output}
 > -1
 > ~~~
 >
-
-> ~~~ {.output}
+> ~~~ {.python}
 > # this is code/arith.py
->import sys
+> import sys
```

Comment:
I always thought the space between the `>` and the rest of the line was necessary in Markdown? It's certainly [there in the spec](https://daringfireball.net/projects/markdown/syntax#blockquote). I suspect if the first line didn't have the extra space, the formatting would be all messed up... but also, it's inconsistent to have a space in the first line but not subsequent lines.